glass/fields.py

```diff
@@ -110,28 +95,33 @@ def iternorm(
 
 
 def cls2cov(
-    cls: Cls, nl: int, nf: int, nc: int
-) -> collections.abc.Generator[npt.NDArray[typing.Any], None, None]:
+    cls: npt.NDArray[np.float64],
```

Comment:
this was correct before: `cls` are a sequence of arrays or sequence of float, not one array

Reply:
So to clarify the expected type is

```python
collections.abc.Sequence[npt.NDArray[np.float64] | collections.abc.Sequence[float]]
```

not

```python
npt.NDArray[np.float64] | collections.abc.Sequence[collections.abc.Sequence[float]]
```

?